tests/test_smoke.py

```diff
@@ -1665,6 +1665,40 @@ def test_core_api():
 class TestStorageWithCredentials:
     """Storage tests which require credentials and network connection"""
 
+    AWS_INVALID_NAMES = [
+        'ab',  # less than 3 characters
+        'abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz1',
+        # more than 63 characters
+        'Abcdef',  # contains an uppercase letter
+        'abc def',  # contains a space
+        '192.168.5.4',  # formatted as an IP address
+        'xn--bucket',  # starts with 'xn--' prefix
+        'bucket-s3alias',  # ends with '-s3alias' suffix
+        'bucket--ol-s3',  # ends with '--ol-s3' suffix
+        '.abc',  # starts with a dot
+        'abc.',  # ends with a dot
+        '-abc',  # starts with a hyphen
+        'abc-',  # ends with a hyphen
```

Comment:
Perhaps adding an invalid AWS case of two adjacent periods `abc..def`?
<img width="448" alt="Screenshot 2023-04-10 at 9 29 07 PM" src="https://user-images.githubusercontent.com/34902420/231056370-7595d7a9-dda5-434d-8147-2c6da621f7b6.png">

Reply:
Done!